app/views/layouts/2020.html.erb

```diff
@@ -9,26 +9,41 @@
     <%= stylesheet_pack_tag '2020main' %>
     <script src="https://kit.fontawesome.com/5dc7d61ac4.js" crossorigin="anonymous"></script>
     <!-- Google tag (gtag.js) -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=G-5QT07E47HT"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
-  gtag('config', 'G-5QT07E47HT');
-</script>
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-5QT07E47HT"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', 'G-5QT07E47HT');
+    </script>
+    <!-- Google Tag Manager -->
+    <script>
+      (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+      new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+      'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+      })(window,document,'script','dataLayer','GTM-PVV4HQM4');
+    </script>
```

Comment:
```suggestion
    <script>
      (
        function(w, d, s, l, i) { 
          w [l] = w [l] || [];
          w [l].push(
              { 'gtm.start': new Date().getTime(),
              event :'gtm.js' }
          );
          var f = d.getElementsByTagName(s) [0],
          j = d.createElement(s),
          dl = l != 'dataLayer' ? '&l=' + l :'';
          j.async = true;
          j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
          f.parentNode.insertBefore(j, f);
        }
      )(window, document, 'script', 'dataLayer', 'GTM-PVV4HQM4');
    </script>
```

Reply:
@ConnorSheremeta do you think this is an improvement?  If not we can go back to the condensed code.

Reply:
I think there are some arguments to be made to keep it condensed and obfuscated, but I think in this case, both to match what was done with the gtag script above and considering it is a relatively short script, this change would be an improvement :+1: